Cleaning + Adaptive links

BellezaEmporium opened the pull request:
Closes #3727 by deleting links that do not work. However, the other Movistar Astra streams still work.

BellezaEmporium committed:
ES : Cleaning + Adaptive links

BellezaEmporium committed:
CN : Beginning cleaning and correcting tvg-ids

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

MapGuy11 committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-clean

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G commented:
Same here, we've got automatic cleaning branch ready. Cleanup can be continued from there

Dum4G commented on channels/cn.m3u:
```diff
@@ -1,14 +1,12 @@
-#EXTINF:-1 tvg-id="ATVAsiaYaZhouDianShiTai.cn" tvg-name="ATV Asia (亞洲電視台)" tvg-country="CN" tvg-language="Chinese" tvg-logo="https://www.hkatv.com/images/logo.png" group-title="",ATV Asia (亞洲電視台)
-https://15813114727637862976168173814320.live.prod.hkatv.com/a1_cbr_hi_1080p.m3u8
-#EXTINF:-1 tvg-id="BlueMeiJuPinDao.cn" tvg-name="Blue 美剧频道" tvg-country="CN" tvg-language="Chinese" tvg-logo="https://parco-zh.github.io/demo/BLUE.png" group-title="",Blue 美剧频道
-http://210.210.155.35:80/session/676c25ba-724e-11e9-834b-8ac28bf32f5a/dr9445/h/h16/02.m3u8
```
works

LaneSh4d0w replied:
Geo-locked then : 
![image](https://user-images.githubusercontent.com/30985701/127784536-41306741-1ddf-4cb9-ae1e-b7e6253b4aaa.png)

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

BellezaEmporium committed:
Update us.m3u

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G commented:
Is it time to reimplement it?

LaneSh4d0w commented:
Give me a few moments to re-examine the Chinese playlist and then we're good to go.

BellezaEmporium committed:
Provide adaptive links + add CCTV+ Channel 2

Channel 2 is mostly retransmitting CGTN programmation in both Chinese and English.

BellezaEmporium committed:
Continue updating CN

BellezaEmporium committed:
Update cn.m3u

BellezaEmporium committed:
Update cn.m3u

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-clean

BellezaEmporium committed:
Update es.m3u

BellezaEmporium committed:
Update es.m3u

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

spotanjo3 commented:
Portugal Channel.. Can't find the right place for this because those are double links which is not need. 

Kuriakos TV (1080p) (Please remove it) One of it is already in the list is (576p)
Bloomberg TV Europe (720p) (Please remove it) 2160p is already in the list.
Bloomberg TV EMEA Live EVENT (720p) (Please remove it and find one) It is not 720. A little blurry.
Kuriakos Cine (1080p) (Please remove it) One of it is already in the list
TV Fátime (1080p) (Please removed it)
TV Fátima (1080p) (Please removed it) Both are the same
Sobrenatural TV (720) (Not 24/7) (Please removed it) One of it is already (1080p) in the list
RPT1 [not 24/7] (Please removed it) Already have RPT1

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G committed:
Merge branch 'master' into LaneSh4d0w-clean

LaneSh4d0w commented:
This will be good for now. Another batch of cleans and adaptive links will come on due time.

BellezaEmporium committed:
Merge branch 'master' into LaneSh4d0w-clean

Dum4G commented on channels/cn.m3u:
```diff
@@ -37,137 +37,121 @@ http://223.110.243.171/PLTV/3/224/3221227204/index.m3u8
+#EXTINF:-1 tvg-id="CGTNEspanol.cn" tvg-name="CGTN Español" tvg-country="CN" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/CGTN.svg/800px-CGTN.svg.png" group-title="News",CGTN Español (720p)
 https://news.cgtn.com/resource/live/espanol/cgtn-e.m3u8
-#EXTINF:-1 tvg-id="CGTN1.cn" tvg-name="CGTN 1" tvg-country="CN" tvg-language="Spanish" tvg-logo="https://astrocontent.s3.amazonaws.com/Images/ChannelLogo/Pos/509_300.png" group-title="",CGTN 1
-https://livees.cgtn.com/1000e/prog_index.m3u8
```
whats up with those prog_index links? All work for me

LaneSh4d0w replied:
If you open the m3u8 file above, those links are in there.

![image](https://user-images.githubusercontent.com/30985701/128784071-02cac8eb-5cda-450f-8168-226fa1d96e0e.png)
